news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,125 @@
+We fix all important regressions as fast as possible because when this point is reached, we can launch the [Beta phase](https://en.wikipedia.org/wiki/Software_release_life_cycle#Beta). Although it depends heavily on the number of found regressions, we try to size this phase to last one month.
+
+For PrestaShop 1.7.7.0, QA team is currently finishing the test campaign.
+
+# Beta period
+
+When the branch reaches a point of maturity, which means only minor or trivial issues remain to be fixed, developers team create a Beta Build using the `1.7.7.x` branch codebase.
+
+This Beta build is provided to anyone and we open a one-month long Beta period where we ask **your** help.
+
+During this one month, we continue testing and fixing the 1.7.7.x (following the stabilization goal) but we know that we can only test and imagine a limited amount of usecases. The community however knows better than us all the possible ways to use PrestaShop to build a business.
+
+So **you can help us** making this release a better and more stable release by testing this Beta build.
+
+What does it mean, "to test a Beta build" ? We do not expect you to go in all pages and click on all buttons !
+What we are interested in is your standard usage of the shop.
+
+- If you use PrestaShop webservices for system integration, please run your integration on this beta build
+- If you build 1.7 modules or themes, please test them on this beta build
+- If you use heavily some of the modified/improved/rewored Back-office pages, please give the new behaviors a try
+- If you customize some parts of the Shop, please test how your customizations behave on this new version
+
+For example if you are a payment module developer, just installing your module on this Beta software,  processing one payment and telling us that everything is running as expected is already a great feedback !
+
+If however you find suspicious behaviors, you can
+ - [report this as a bug on GitHub](https://github.com/PrestaShop/PrestaShop/issues) (read [how to report issues](https://devdocs.prestashop.com/1.7/contribute/contribute-reporting-issues/))
+ - submit a bug fix by creating a [pull request](https://github.com/PrestaShop/PrestaShop/compare) (read the [contribution guidelines](https://devdocs.prestashop.com/1.7/contribute/contribution-guidelines/))
+
+# Release Candidate
+
+When Beta period ends, we consider that all known bugs for this release are now registered into the Kanban. So the aim is clear: fix it all and we can ship.
+
+We fix the remaining bugs (or merge the bug fixes submitted by the community) and maintainers deliver a Release Candidate Build using the `1.7.7.x` branch codebase. This will be the Release Candidate 1 also known as RC1.
+
+This Build is provided to everyone, and we then start the **timer**. We wait for one week. During this week we continue testing and exploring the Build, trying to find anything that would not have been detected earlier and we hope that community will do the same.
```

Comment:
```suggestion
This Build is provided to everyone, and we then start the **timer**. We wait for one week. During this week we continue testing and exploring the Build, trying to find anything that would not have been detected earlier and the community must do the same.
```

Reply:
Maybe must is a little too much, maybe should is better, wdyt ?

Reply:
Should looks good 👍